src/App.jsx

```diff
@@ -14,33 +16,95 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+export const App = () => {
+  const [visibleGoods, setVisibleGoods] = useState(goodsFromServer);
+  const [actions, setActions] = useState({ sort: null, reversed: false });
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+  const SortA = () => {
+    const sorted = [...visibleGoods].sort((good1, good2) => {
+      return good1.localeCompare(good2);
+    });
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+    setVisibleGoods(actions.reversed ? sorted.reverse() : sorted);
+    setActions({ sort: 'alphabetical', reversed: actions.reversed });
+  };
 
-      <button type="button" className="button is-danger is-light">
-        Reset
-      </button>
-    </div>
+  const SortByLength = () => {
```

Comment:
why functions naming in PascalCase. for functions use camelCase.  only for components functions use PascalCase